finagle-netty4/src/main/scala/com/twitter/finagle/netty4/ssl/server/Netty4ServerSslConfigurations.scala

```diff
@@ -65,6 +61,8 @@ private[finagle] object Netty4ServerSslConfigurations {
           cert <- new X509CertificateFile(certFile).readX509Certificate()
           chain <- new X509CertificateFile(chainFile).readX509Certificates()
         } yield SslContextBuilder.forServer(key, cert +: chain: _*)
+      case KeyCredentials.KeyManagerFactory(keyManagerFactory)          =>
```

Comment:
Can you add a note to the comments for this method that indicates that unlike Cert/Key Cert/Key/Chain, that using KeyManagerFactory does not ensure that the certificates contained within the KeyManagerFactory are not expired.